public/landing.html

```diff
@@ -0,0 +1,366 @@
+				break;
+
+			case "treeCount": document.getElementById("treeCountCust").style.display = "block";
+				break;
+
+			case "tenantLead": document.getElementById("tenantLeadCust").style.display = "block";
+				break;
+
+			default: break;
+		}
+	}
+	function validURLTreeMap(str) {
+		var expression = /[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)?/gi;
+		var regex = new RegExp(expression);
+		document.getElementById("error").innerHTML = "";
+		if (str.match(regex)) {
+			const lastSegment = str.split("/").pop();
+			updateUserAttribute(lastSegment)
+		}
+		else {
+			updateUserAttribute(str)
+		}
+	}
+
+	function updateUserAttribute(userslug) {
+		fetch(`https://app.plant-for-the-planet.org/app/profiles/${userslug}`)
+			.then((response) => {
+				if (!response.ok) {
+					throw new Error("Invalid User");
+				} else return response.json();
+			})
+			.then((data) => {
+				
+				// let theme = document.getElementById("treeMap").getAttribute("theme");
+				// let primaryColor = document.getElementById("treeMap").getAttribute("primarycolor");
+				// let circleBGColor = document.getElementById("treeMap").getAttribute("circlebgcolor");
+				// let community = document.getElementById("treeMap").getAttribute("community");
+				// let locale = document.getElementById("treeMap").getAttribute("locale");
+
+				document.getElementById("treemapp").innerHTML = `<tree-map user=${data.slug}  id="treeMap"> </tree-map>`;
+			})
+			.catch((error) => {
+				document.getElementById("error").innerHTML = "Invalid Profile URL";
+			});
+	}
+
+	function validURLTreeProf(str) {
+		var expression = /[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)?/gi;
+		var regex = new RegExp(expression);
+		document.getElementById("error-prof").innerHTML = "";
+		if (str.match(regex)) {
+			const lastSegment = str.split("/").pop();
+			updateUserAttributeProf(lastSegment)
+		}
+		else {
+			updateUserAttributeProf(str)
+		}
+	}
+
+	function updateUserAttributeProf(userslug) {
+		fetch(`https://app.plant-for-the-planet.org/app/profiles/${userslug}`)
+			.then((response) => {
+				if (!response.ok) {
+					throw new Error("Invalid User");
+				} else return response.json();
+			})
+			.then((data) => {
+				
+				// let theme = document.getElementById("treeMap").getAttribute("theme");
+				// let primaryColor = document.getElementById("treeMap").getAttribute("primarycolor");
+				// let circleBGColor = document.getElementById("treeMap").getAttribute("circlebgcolor");
+				// let community = document.getElementById("treeMap").getAttribute("community");
+				// let locale = document.getElementById("treeMap").getAttribute("locale");
+
+				document.getElementById("treeProf").innerHTML = `<tree-profile user=${data.slug}  id="treeProfile"> </tree-profile>`;
+			})
+			.catch((error) => {
+				document.getElementById("error-prof").innerHTML = "Invalid Profile URL";
+			});
+	}
```

Comment:
This looks like a duplicate of the code before?

Reply:
Looks similar with different ID's, this one in particular is for tree-profile.